pkg/cli/clisqlshell/sql.go

```diff
@@ -1235,11 +1235,9 @@ func (c *cliState) doHandleCliCmd(loopState, nextState cliStateEnum) cliStateEnu
 
 	case `\copy`:
 		c.exitErr = c.runWithInterruptableCtx(func(ctx context.Context) error {
-			return c.beginCopyFrom(ctx, c.concatLines)
+			// Strip out the starting \ in \copy.
+			return c.beginCopyFrom(ctx, line[1:])
 		})
-		if !c.singleStatement {
```

Comment:
why don't we need this?

Reply:
ah we do, i don't know where i copy pasta'd this from tho. i also swear my initial prototype worked, but alas, here we are!